examples/rag_report_demo_why_some_customers_donot_like_pizza.sql

```diff
@@ -0,0 +1,205 @@
+-- The demo of AI RAG by generate a why customer don't like pizza report.
+--
+-- The input is 3 records of customer comments to pizza food
+-- the output is ai result table with 3 columns:send_message, chat_completion,  final_report
+--    the final report columns is markdown formated business analysis report to why customer don't like pizza
+
+-- the process/tools:
+	--- vector database: pgai container for run an store the input, output data
+    --- ai model service: openai 'gpt-4o-mini','text-embedding-3-small'
+    --- the rag concept of process
+
+-- step 0. Check environments(disabled by default)
+-- make sure pgai docker is running normally
+-- check openai connection
+--select * from openai_list_models()  olm
+--where id in ('gpt-4o-mini','text-embedding-3-small');
+--
+---- check the gpt-4o-mini , text-embedding-3-smal work well.
+--with 
+--tmp as (select 'what is your name?' as message )
+--, test_result as (
+--	select
+--		message, 
+--		openai_chat_complete(
+--			'gpt-4o-mini',
+--			jsonb_build_array(
+--				jsonb_build_object('role', 'user', 'content', message)
+--			)) as chat, 
+--		openai_embed('text-embedding-3-small',message) as embed
+--	from tmp)
+--select * from test_result;
+
```

Comment:
I think these steps could be removed as they're also in the main README 🤔 

```suggestion
```

Reply:
Sure. Done